### PR TITLE
4.4-dev

### PR DESCRIPTION
**Pull request by webmasterab**
Pull Request for Issue # .

This is new to me.
If I did it wrong please let me know.

### Summary of Changes
I changed fixed translation to translatable strings

### Actual result BEFORE applying this Pull Request

In localAdapter.php there was untranslatable text

### Expected result AFTER applying this Pull Request

In localAdapter.php the text is now translatable using string

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: language/en-GB/com_media.ini [hunk 1]
```diff
@@ -31,6 +31,18 @@ COM_MEDIA_CURRENT_PROGRESS="Current progress"
 COM_MEDIA_DECREASE_GRID="Decrease grid size"
 COM_MEDIA_DELETE_ERROR="Error deleting the item."
 COM_MEDIA_DELETE_SUCCESS="Item deleted."
+COM_MEDIA_DELETE_NOT_POSSIBLE="Delete not possible!"
```
Comment (Quy) [inline language/en-GB/com_media.ini]: Please sort keys in alpha order.

Comment (webmasterab) [reply language/en-GB/com_media.ini]: Thanks for this tip
I'll see if I can change that here.

**Review by webmasterab — COMMENTED** [review]
I'm trying to put it in order but can't find a good way yet.
Working on it

**Comment by webmasterab** [comment]
It turns out to be more difficult than I thought to get it right.
Close it and insert a new one.

**Pull request closed without merging**